src/components/Navibar.js

```diff
@@ -3,6 +3,7 @@ import { Navbar, NavbarToggler, NavbarBrand,
        Nav, NavItem, NavLink, Collapse } from 'reactstrap';
 import { Home, Support, Contact, Goods } from './constant';
 import './home.css';
+import logo from './logo/AgroeasyLogo.png';
```

Comment:
all images are saved in the `images` folder. delete the `logo` folder and move `AgroeasyLogo.png` to the images folder. There is no need to import the image as well, just provide the path directly to the `src` using it